media/js/firefox/australis/browser-tour.js

```diff
@@ -504,8 +524,13 @@ if (typeof Mozilla == 'undefined') {
             buttons,
             options
         );
-
-        gaTrack(['_trackEvent', 'Tour Interaction', 'Later', 'The new Forget Button']);
+        
+        window.dataLayer = window.dataLayer || [];
```

Comment:
In cases where the dataLayer.push is not reliant upon user interaction, it is possible for the object to be pushed to the dataLayer before the dataLayer is instantiated.  This is why it is necessary for an asynchronous instantiation of the dataLayer.  

@alexgibson From my understanding, this doorhanger pops up on page load. Is this correct? If this is reliant on user interaction it can be removed.